common/src/main/resources/wynntils.mixins.json

```diff
@@ -45,8 +46,11 @@
   ],
   "compatibilityLevel": "JAVA_17",
   "injected_interfaces": {
+    "net/minecraft/class_1297": [
+      "com/wynntils/models/entities/WynntilsCustomGlowEntityProperty"
+    ],
     "net/minecraft/class_437": [
-      "com/wynntils/gui/widgets/TextInputBoxWidget"
+      "com/wynntils/screens/base/TextboxScreen"
```

Comment:
@magicus

Reply:
Yeah, look at that! :) This is not needed. That is only if you really want the actual target class to have a new interface. We are only using this as a way to get around the java compiler otherwise saying "no there is no such method in Screen/ItemStack/Entity". I'm not adding it for ItemAnnotations and it works just as fine.

Reply:
But I'll clean that up as well.